integration-cli/docker_cli_images_test.go

```diff
@@ -222,12 +201,8 @@ func (s *DockerSuite) TestImagesEnsureOnlyHeadsImagesShown(c *check.C) {
 	intermediate := strings.TrimSpace(split[5][7:])
 
 	out, _ = dockerCmd(c, "images")
-	if strings.Contains(out, intermediate) {
-		c.Fatalf("images shouldn't show non-heads images, got %s in %s", intermediate, out)
-	}
-	if !strings.Contains(out, head[:12]) {
-		c.Fatalf("images should contain final built images, want %s in out, got %s", head[:12], out)
-	}
+	c.Assert(out, checker.Not(checker.Contains), intermediate, check.Commentf("images shouldn't show non-heads images, got %s in %s", intermediate, out))
+	c.Assert(out, checker.Contains, head[:12], check.Commentf("images should contain final built images, want %s in out, got %s", head[:12], out))
```

Comment:
No need for out here or the head[:12]
I'd probably just move this comment to a normal go comment above the assert.